examples/berad/pkg/state-transition/state_processor.go

```diff
@@ -58,6 +58,9 @@ type StateProcessor[
 	ExecutionPayloadHeaderT ExecutionPayloadHeader,
 	ForkT interface {
 		New(common.Version, common.Version, math.Epoch) ForkT
+		GetPreviousVersion() common.Version
+		GetCurrentVersion() common.Version
+		GetEpoch() math.Epoch
```

Comment:
define a type for this interface. and put in the same file as where the other types used are defined (`ExecutionPayloadHeader`, `ForkData`, etc)

Reply:
Good point, done.